src/test/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterTest.java

```diff
@@ -51,33 +58,166 @@ public void testAccept() throws Exception {
     @Test
     public void testMultiChangesOnOneFileOne() throws Exception {
         final String fileName = getPath("MultiChangesOnOneFilePatch.txt");
-        final SuppressionPatchFilter filter = createSuppressionPatchFilter(fileName);
-        final LocalizedMessage message = new LocalizedMessage(4, 1, null, "msg", null,
-                SeverityLevel.ERROR, null, getClass(), null);
-        final AuditEvent ev = new AuditEvent(this, "Update.java", message);
-        assertTrue(filter.accept(ev),
-                "Audit event should be rejected when there are no matching patch filters");
+        final FileInputStream ins = new FileInputStream(fileName);
+        final UnifiedDiff diff = UnifiedDiffReader.parseUnifiedDiff(ins);
+        assertEquals(1, diff.getFiles().size());
+        final UnifiedDiffFile file1 = diff.getFiles().get(0);
+        assertNull(file1.getDiffCommand());
+        assertEquals("Origin.java", file1.getFromFile());
+        assertEquals("Update.java", file1.getToFile());
+        assertEquals(2, file1.getPatch().getDeltas().size());
+        assertNull(diff.getTail());
     }
 
     @Test
-    public void testMultiChangedFilesOnOnePatch() throws Exception {
+    @Ignore("https://github.com/java-diff-utils/java-diff-utils/issues/83")
+    public void testMultiChangedFilesOnOnePatchOne() throws Exception {
+        final String fileName = getPath("problem_diff_issue51.diff");
```

Comment:
what is issue 51 in the filename? Shouldn't it be issue 83?

Reply:
"problem_diff_issue51.diff" is a test file from java-diffs-utils UT, so I do not change its name.